src/app/folder/folderlist.component.ts

```diff
@@ -140,13 +144,58 @@ export class FolderListComponent {
     private _getLevel = (node: FolderCountEntry) => node.folderLevel;
     private _isExpandable = (node: FolderCountEntry) => node.isExpandable ? true : false;
 
-    allowDropToFolder(event, folderId: number): void {
+    /**
+     * Folderlist entry is 48 pixels, so if mouse is in the upper region suggest dropping above,
```

Comment:
48 is a bit of a magic number here.

Is it possible to fetch this number of pixels straight from the Folderlist component? This would make us more future-proof.

Reply:
Now getting the tree node height from DOM.

Reply:
This makes the comment outdated – but it's great! Is it possible to do the same for the `7`?

Reply:
No, since that's a more like a choice we have to make. Dropping within the 7 upper or lower will result in the folder being dropped above or below. So that's nothing that can be read from an element. It's more like a constant to be adjusted if the space for drop above/below needs to be increased / reduced.

Reply:
Alright, makes sense :)